neutron/assetlist.json

```diff
@@ -1451,6 +1451,117 @@
+      "address": "neutron10dxyft3nv4vpxh5vrpn0xw8geej8dw3g39g7nqp8mrm307ypssksau29af",
+      "base": "cw20:neutron10dxyft3nv4vpxh5vrpn0xw8geej8dw3g39g7nqp8mrm307ypssksau29af",
+      "name": "Astrovault AXV (Neutron)",
+      "display": "AXV",
+      "symbol": "AXV",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/axv.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/axv.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/axv.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/axv.svg"
+        }
+      ],
+      "socials": {
+        "website": "https://astrovault.io/",
+        "twitter": "https://x.com/axvdex"
+      },
+      "coingecko_id": "astrovault"
+    },
+    {
+      "description": "IBC Axelar wbtc-satoshi through axelar-dojo-1 transfer/channel-2",
+      "denom_units": [
+        {
+          "denom": "ibc/DF8722298D192AAB85D86D0462E8166234A6A9A572DD4A2EA7996029DF4DB363",
+          "exponent": 0,
+          "aliases": [
+            "uusdc"
+          ]
+        },
+        {
+          "denom": "axlwbtc",
+          "exponent": 8
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/DF8722298D192AAB85D86D0462E8166234A6A9A572DD4A2EA7996029DF4DB363",
+      "name": "Wrapped Bitcoin (Axelar)",
+      "display": "axlwbtc",
+      "symbol": "axlWBTC",
+      "coingecko_id": "axlwbtc",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "axelar",
+            "base_denom": "wbtc-satoshi",
+            "channel_id": "channel-78"
+          },
+          "chain": {
+            "channel_id": "channel-2",
+            "path": "transfer/channel-2/wbtc-satoshi"
+          }
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/axelar/images/wbtc.png"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/axelar/images/wbtc.png",
+          "theme": {
+            "primary_color_hex": "#41394d"
+          }
+        }
+      ]
+    },
+    {
+      "description": "Astrovault xATOM on Neutron Chain",
+       "denom_units": [
+         {
+           "denom": "cw20:neutron1vjl4ze7gr32lar5s4fj776v70j4ml7mlt4aqln2hwgfhqjck8xwqfhx8vj",
+           "exponent": 0
+         },
+         {
+           "denom": "xATOM",
+           "exponent": 6
+         }
+       ],
+       "type_asset": "cw20",
+       "address": "neutron1vjl4ze7gr32lar5s4fj776v70j4ml7mlt4aqln2hwgfhqjck8xwqfhx8vj",
+       "base": "cw20:neutron1vjl4ze7gr32lar5s4fj776v70j4ml7mlt4aqln2hwgfhqjck8xwqfhx8vj",
+       "name": "Astrovault xATOM (Neutron)",
+       "display": "xATOM",
+       "symbol": "xATOM",
```

Comment:
Because this symbol has lowercase letters, please define the `traces` and how it derives from ATOM